scripts/setup.sh

```diff
@@ -5,7 +5,10 @@ SCRIPT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 
 stack_resolver() {
   local DEFAULT_RESOLVER=lts
-  local CONFIGURED=$( sed -rn 's/^resolver:\s*(\S+).*$/\1/p' "$1" 2>/dev/null )
+  local CONFIGURED
+
+  CONFIGURED=$(awk '{if ($1 == "resolver:") {print $2}}' "$1") \
```

Comment:
Very cool to see how this works, I've never used awk actually. Just tried this in my terminal and sure enough it extracts the value of resolver when present or empty string when absent.